cupy/cuda/compiler.py

```diff
@@ -172,12 +176,16 @@ def _jitify_prep(source, options, cu_path):
     # (NVIDIA/jitify#79).
 
     try:
-        name, options, headers, include_names, hdr_map = jitify(
+        name, options, headers, include_names = jitify(
             source, options, _jitify_header_source_map)
     except Exception as e:  # C++ could throw all kinds of errors
-        raise JitifyException(str(e))
+        cex = CompileException(str(e), source, cu_path, options, 'jitify')
+        dump = _get_bool_env_variable(
+            'CUPY_DUMP_CUDA_SOURCE_ON_ERROR', False)
+        if dump:
+            cex.dump(sys.stderr)
+        raise JitifyException(str(cex))
     assert name == cu_path
```

Comment:
This line ensures that the upstream change (NVIDIA/jitify#82) is correctly behaving.